rest/route/commit_queue.go

```diff
@@ -0,0 +1,47 @@
+package route
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/evergreen-ci/evergreen/rest/data"
+	"github.com/evergreen-ci/evergreen/rest/model"
+	"github.com/evergreen-ci/gimlet"
+	"github.com/pkg/errors"
+)
+
+type commitQueueGetHandler struct {
+	project string
+
+	sc data.Connector
+}
+
+func makeGetCommitQueueItems(sc data.Connector) gimlet.RouteHandler {
+	return commitQueueGetHandler{
+		sc: sc,
+	}
+}
+
+func (cq commitQueueGetHandler) Factory() gimlet.RouteHandler {
+	return &commitQueueGetHandler{
+		sc: cq.sc,
+	}
+}
+
+func (cq commitQueueGetHandler) Parse(ctx context.Context, r *http.Request) error {
+	cq.project = gimlet.GetVars(r)["project_id"]
+	return nil
```

Comment:
Should we return a 404 if the project doesn't exist?

Reply:
done.